docs/source/private/prep_and_debug.rst

```diff
@@ -0,0 +1,51 @@
+.. _prep_and_debug:
+
+***********************************************
+FAQ: Preparing and Debugging OpenFE Simulations
+***********************************************
+
+This FAQ contains a list of issues you may encounter when preparing and
+running simulations with OpenFE. This list will be updated as feedback
+is provided by benchmark partners.
+
+
+1. Ligand Alignment
+*******************
+
+Ligands in a series should be pre-aligned when pass to OpenFE tools.
+This means that regions of the ligand that should be part of a common
+core, should have the same coordinates.
+
+In these benchmarks we will use the Kartograf atom mapper. Any ligand
+regions which do not align in 3D space will be unmapped and treated
+as unique atoms in the alchemical topology.
+
+
+2. Protein Structures Ready for Simulation
+******************************************
+
+OpenFE does not run a very long equilibration during the RBFE calculation.
+As such, a protein structure with clashes will likely fail with the Protocol
+reporting "NaN" errors.
+
+You should make sure that your protein structure is ready for MD simulation
+before passing it to OpenFE.
+
+
+3. All My Complex Edges Fail But Not The Ligands
+************************************************
+
+This indicates a likely issue with the protein structure, either a clash
+with the ligand, or a clash within the protein structure itself.
+
+To check this we suggest running a conventional MD simulation and seeing if the system remains stable.
+You can use the `OpenFE MD Protocol for this purpose <https://docs.openfree.energy/en/stable/tutorials/md_tutorial.html>`.
```

Comment:
```suggestion
You can use the `OpenFE MD Protocol for this purpose <https://docs.openfree.energy/en/stable/tutorials/md_tutorial.html>`_.
```

Reply:
I'm not sure what's going on with the formatting here, either in the original or in Hannah's suggested change.

Reply:
Maybe it's a broken hyperlink?